mrtt-ui/src/views/Auth/SignupForm.js

```diff
@@ -38,9 +48,9 @@ const SignupForm = () => {
     formState: { errors }
   } = useForm({ resolver: yupResolver(validationSchema), defaultValues: formDefaultValues })
 
-  const signUp = (formData) => {
+  const signUp = ({ email, name, password }) => {
     axios
-      .post(authUrl, { user: formData })
+      .post(authUrl, { user: { email, name, password } })
```

Comment:
didnt want to send the extra confirm password input value incase the server gets strict about it in the future.